aten/src/ATen/native/sparse/cuda/SparseCUDATensorMath.cu

```diff
@@ -268,7 +268,7 @@ Tensor& add_out_dense_sparse_cuda(Tensor& r_, const Tensor& dense, SparseTensorR
     r_.resize_as_(dense);
     r_.copy_(dense);
   } else {
-    AT_CHECK(r_.is_contiguous(), "CUDA dense-sparse addition known bug");
+    AT_CHECK(r_.is_contiguous(), "add: CUDA dense-sparse addition does not work; shout if you need it");
```

Comment:
From the check, it seems that `CUDA dense-sparse addition` works in contiguous case. So the error message is wrong.

Also, since we have this, why the line below?

Reply:
It didn't work a year ago: https://github.com/pytorch/pytorch/issues/1521

Maybe it works now? Have to check

Reply:
Oh I see what you're saying.